codersdk/templateversions.go

```diff
@@ -8,6 +8,7 @@ import (
 	"net/http"
 	"time"
 
+	"github.com/coder/terraform-provider-coder/provider"
```

Comment:
I guess it should be `codersdk` import?

Reply:
Could you elaborate please?
I wanted to ensure the provider and the server both use the same strings.

Reply:
This is probably about the encapsulation so that `codersdk` just defines the entire domain as it is the API, and does not depend on properties/strings defined in external libraries. If the library value changes, this will be a breaking change in the API.

Reply:
OK, that's fair. I'll revert this change.